### DISCUSSION TITLE
Tâche planifié

### DISCUSSION THREAD

**Discussion by ArthurBergere** [dadoonet/fscrawler#1621] [Q&A]
Bonjour,

Savez-vous s'il y a moyen de faire qui se lance de manière planifié, pour lancer fscrawler et aussi le fermer au bout de x temps ? 
Moi j'ai un script pour lancer toutes mes indexations, j'ai aussi une tâche planifié, mais je n'arrive pas à fermer toute mes fenêtres liés à fscrawler.

Merci d'avance :)

**Comment by dadoonet** [comment]
Bonjour


Avec l'option `--loop 1`, FSCrawler se fermera automatiquement après un scan.
Voir https://fscrawler.readthedocs.io/en/latest/admin/cli-options.html#loop
